flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/api/TableITCase.scala

```diff
@@ -104,14 +104,35 @@ class TableITCase(tableEnvName: String, isStreaming: Boolean) extends TestLogger
 
   @Test
   def testExecuteWithUpdateChanges(): Unit = {
```

Comment:
Is this the same with `TableEnvironmentITCase#testExecuteSelectWithUpdateChanges`?